.gitignore

```diff
@@ -14,3 +14,6 @@ bin/
 
 # Goreleaser Output
 dist/
+
+# VSCode setup and plugins
+.vscode/ 
```

Comment:
This should be part of a VS Code user's global `.gitignore` and, IMHO, has nothing to do with the project.